lighthergm/R/gof_lighthergm.R

```diff
@@ -273,7 +286,7 @@ gof_lighthergm <- function(net,
           seed_edgelist = network::as.edgelist(base_network),
           coef_within_block = coef_within_block,
           output = 'network',
-          ergm_control = ergm_control,
+          ergm_control = ergm_control_sim,
```

Comment:
This might be out of the scope of this PR, but It would be better to explicitly write down all the argument names here like `arg = hoge`
https://github.com/sansan-inc/lighthergm/blob/d210cb1e84ad5213965e7f0fae9805923a16f4f7/lighthergm/R/gof_lighthergm.R#L262-L279